src/sentry/feedback/usecases/create_feedback.py

```diff
@@ -257,7 +257,10 @@ def create_feedback_issue(event, project_id: int, source: FeedbackCreationSource
         auto_ignore_spam_feedbacks(project, issue_fingerprint)
     metrics.incr(
         "feedback.create_feedback_issue.produced_occurrence",
-        tags={"referrer": source.value},
+        tags={
+            "referrer": source.value,
+            "client_source": event["contexts"]["feedback"].get("source"),
```

Comment:
is `event["contexts"]["feedback"].get("source")` different from `source.value`?

Reply:
Yeah -- the feedback context is SDK-defined and passed from there. Think source is required, but using .get in case of typescript 

The `source` in this function is a FeedbackCreationSource enum, which is chosen by Sentry backend before calling `create_feedback_issue`

Reply:
ahhh gotcha, makes sense